package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/Yolean/kafka-cache#readme",
   "devDependencies": {
+    "@types/node-rdkafka": "github:Yolean/node-rdkafka-types",
```

Comment:
The reason I'm adding this, despite kafka-cache not being TypeScript (yet), is that I think we should consistently do that when removing node-rdkafka from package.json, to indicate that the dependency is still there.